index.html

```diff
@@ -1,210 +1,106 @@
-                               and their families.</li>
-                    </ul></p>
-				</article>
-				
-				<!--<article>
-					<h2>Security Assistant at University of Miami, Coral Gables, FL</h2>
-					<p class="subDetails">2008-2009</p>
-					<p><ul>
-                           <li>Confronted and excluded any undesirable individuals from the residential colleges.</li>
-                           <li>Coordinated emergency responses (health, fire, bomb threats, discipline inidents)</li>
-                           <li>Maintained accurate and legible documentation of events.</li></ul></p>
-				</article>
-				
-				<article>
-					<h2>IT User Assistant at University of Miami, Coral Gables, FL</h2>
-					<p class="subDetails">2007-2008</p>
-					<p><ul>
-                           <li>Monitored omputer labs on the University of Miami campus.</li>
-                           <li>Ensured integrity of equipment and materials as well as provided assistance to students.</li></ul></p>
-				</article> -->
-			</div>
-			<div class="clear"></div>
-		</section>
-		
-		
-		<section>
-			<div class="sectionTitle">
-				<h1>Languages</h1>
-			</div>
-			
-			<div class="sectionContent">
-				<ul class="keySkills">
-					<li><span class="language">English</span>, Native</li>
-                    <li><span class="language">Papiamento</span>, Native</li>
-					<li><span class="language">Spanish</span>, Proficient</li>
-					<li><span class="language">Dutch</span>, Intermediate</li>
-					
-				</ul>
-			</div>
-			<div class="clear"></div>
-		</section>
-		
-         <section>
-			<div class="sectionTitle">
-				<h1>Volunteer Experience</h1>
-			</div>
-			
-			<div class="sectionContent">
-				<article>
-					<h2></h2>
-					<p class="subDetails"></p>
-					<p>Tutor with Reading Partners in Sunnyvale.  In Austin I volunteered regularly at a dog shelter as well as events including SXSW, Austin Marathon, Thundercloud Subs Turkey Trot, and Wanderlust Festival.</p>
-				</article>
-				
-			</div>
-			<div class="clear"></div>
-		</section>
-		
-		
-		
-	</div>
-</div>
-<script type="text/javascript">
-var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
-document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
-</script>
-<script type="text/javascript">
-var pageTracker = _gat._getTracker("UA-3753241-1");
-pageTracker._initData();
-pageTracker._trackPageview();
-</script>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <title>FCC Personal Portfolio Project</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous"><link rel="stylesheet" href="./style.css">
+</head>
+<body>
+<!-- partial:index.partial.html -->
+<!DOCTYPE html>
```

Comment:
What's going on here? Page just starts over? Did this come from the right place?